python/docs/source/index.rst

```diff
@@ -0,0 +1,36 @@
+CCAO Python package
+===================
```

Comment:
I don't love that we have some duplicate content between `ccao/python/README.md` and this docs index file, but I couldn't think of a clean way to have them share some content but not others.